dynomitemanager-common/src/main/java/com/netflix/nfsidecar/config/CommonConfig.java

```diff
@@ -55,12 +55,20 @@
      */
     @DefaultValue("8101")
     @PropertyName(name = "dyno.peer.port")
-    public int getStoragePeerPort();
+    public int getDynomitePeerPort();
 
     @DefaultValue("8102")
     @PropertyName(name = "dyno.port")
     public int getDynomitePort();
 
+    @DefaultValue("22222")
+    @PropertyName(name = "dyno.stats.port")
+    public int getDynomiteStatsPort();
+
+    @DefaultValue("true")
```

Comment:
Do we need to have this `false` or `true` in the OSS? It would be another property for us so there is an overhead, but it might be hard to debug issues.